modules/convert.sh

```diff
@@ -0,0 +1,30 @@
+function convert {
+  checkargn $# 2
+  inputFile=$1
+  outputFile=$2
+  if [[ "$inputFile" != "" ]] && [[ "$outputFile" != "" ]]; then
+    ffmpeg -i $inputFile $outputFile -hide_banner
+    status=$?
+    if [ "$status" == 0 ]; then
+      echo "$inputFile has been successfully converted to $outputFile"
+    else
```

Comment:
I'm not sure if ffmpeg always returns a 0 on success.

Searching online, it seems ffmpeg does not reliably return 0 on success.

When I was personally testing the conversion, ffmpeg gave me an error but it still returned 0 (leading to the success message echoing).

So I don't think you can use `status=$?` to check success or failure.

Reply:
Had to do some deep diving on this one. It seems to work much better now. I was able to get mp4->mp3, mp4->gif, and wmv, and more. I tried mp3->webm and that worked too. I hope we can't find anymore false positives. The -max_error_rate 0.0 param seemed to help a bunch.